tfx/orchestration/experimental/kubernetes/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
```

Comment:
Per the comment in the README, could you remove this file, adding instructions to that readme instead?

Reply:
Removed.